src/EndpointV2/EndpointV2Middleware.php

```diff
@@ -394,4 +402,23 @@ private function resolveAccountId(): ?string
 
         return $identity->getAccountId();
     }
+
+    private function hookAccountIdMetric($accountId, &$command)
```

Comment:
question: can we make a generic method on the MetricsBuilder class that will handle this and other things like it i.e. accountidendpointmetric?

Reply:
Yes, I can take a look and see if it fits better there.